### ISSUE TITLE
MetaMorpheus for middle-down analysis

### ISSUE THREAD

**Issue by piroman999** [smith-chem-wisc/MetaMorpheus#2106]
Hello, I am working with very problematic sample type, a mixture of nonspecific peptides of length from several to ~100 AA. I have noticed that MetaMorpheus have some kind of deconvolution applied, which may be very useful. How to tweak search parameters to avoid extremally long run times?

**Comment by trishorts** [comment]
so, you are using no protease at all? can you tell me how complicated the sample is? is there a way to reduce the size of the search database reasonably?

**Comment by piroman999** [comment]
Yes, no protease at all. The only idea for search space reduction that I have is to make a tryptic digest of the sample, run it, perform semitryptic or nonspecific search on obtained (much shorte I hope) peptides, and use the results to make a database of limited size to search full-length sample data.

**Comment by trishorts** [comment]
I think that might be your best bet for now. We have a nascent effort to do a top-down biomarker style search that could ID longer peptides and so forth.
